cpp/models/abm/time.h

```diff
@@ -26,51 +26,52 @@ namespace abm
 {
 
 /**
- * a duration of time.
- * resolution 1 second.
+ * @brief A duration of time.
+ * Resolution 1 second.
  */
 class TimeSpan
 {
 public:
     /**
-     * default ctor, uninitialized.
+     * @brief Default ctor, uninitialized.
      */
     TimeSpan() = default;
 
     /**
-     * creates a TimeSpan that represents a number of seconds.
+     * @brief Creates a TimeSpan that represents a number of seconds.
+     * @param[in] seconds The number of seconds.
      */
     explicit TimeSpan(int seconds)
         : m_seconds(seconds)
     {
     }
 
     /**
-     * length of time in days.
+     * @brief Length of time in days.
      */
```

Comment:
In this file, there are no `return ...`comments. For my taste, they are not required here (and in all the other getters of the codebase). Please discuss.

Reply:
Removed the return comments for getters